pkg/users/local.js

```diff
@@ -1193,7 +1193,16 @@ function AccountExpiration() {
     $('#account-expiration-input').datepicker({
         autoclose: true,
         todayHighlight: true,
-        format: 'yyyy-mm-dd'
+        language: (function() {
+            try {
+                if (cockpit.language && cockpit.language !== 'en')
+                    require('bootstrap-datepicker/dist/locales/bootstrap-datepicker.' +
+                            cockpit.language.replace('_', '-') + '.min');
+            } catch (ex) {
+                console.warn('Setting datepicker locale failed', ex);
+            }
+            return cockpit.language.replace('_', '-');
+        })()
```

Comment:
This doesn't seem to work: When I switch to German and pick a date from dialog, it gets shown in German format, and then it complains that it is invalid:

![account-exp](https://user-images.githubusercontent.com/200109/41860728-9fcc398a-789f-11e8-8aac-7d8cc5a8b243.png)

Reply:
Going over it again it seems that the dates all sort of depend on each other to be formatted a certain way in this page. 

I think I'll remove the paches involving the users page, and keep that for a later pr.